server/game/cards/01_SOR/events/ForceThrow.ts

```diff
@@ -0,0 +1,35 @@
+import { log } from 'console';
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { TargetMode, WildcardZoneName, WildcardCardType, Trait, EventName } from '../../../core/Constants';
+
+export default class ForceThrow extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '1705806419',
+            internalName: 'force-throw',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Choose a player. That player discards a card from their hand',
+            targetResolver: {
+                mode: TargetMode.Player,
+                immediateEffect: AbilityHelper.immediateEffects.discardCardsFromOwnHand({ amount: 1 }),
+            },
+            then: (thenContext) => ({
```

Comment:
This should probably be an `ifYouDo: (ifYouDoContext)`, which will cause the second part of the ability to not happen if the discard doesn't happen. We should add a test where the target player's hand is empty to confirm that nothing breaks.